CRM/RcBase/Test/UtilsBasic.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * Contains helper functions for basic unit-testing
+ */
+class CRM_RcBase_Test_UtilsBasic
+{
+    /**
+     * Gets a restricted property
+     *
+     * @param mixed $object Object with restricted property
+     * @param string $property Name of property
+     *
+     * @return mixed
+     *
+     * @throws \ReflectionException
+     */
+    public static function getProtectedProperty(&$object, string $property)
```

Comment:
Is this function used anywhere? I can only see the testcase.

Reply:
There is a test involving a Form_Base class, which is not submitted yet. This function is used there. This is not an essential function like can't test without it. Also Reflections are evil and tests shouldn't call invoked methods and check internal variables. Maybe when testing Base classes that could be useful.

Thats was the original reason, but I can let these methods go.
If you think they will be no use later and we should not rely on Reflection classes I'm happy to leave these methods behind.